schema_based_on_diagram.sql

```diff
@@ -0,0 +1,56 @@
+/*CReat new databse */
+
+CREATE DATABASE clinic;
+
+CREATE TABLE patients (
+  id INT GENERATED BY DEFAULT AS IDENTITY ,
+  name VARCHAR,
+  date_of_birth DATE,
+  PRIMARY key (id)
+);
+
+CREATE TABLE medical_histories (
+  id INT GENERATED BY DEFAULT AS IDENTITY,
+  admitted_at TIMESTAMP,
+  patient_id INT REFERENCES patients (id),
+  status VARCHAR,
+  PRIMARY KEY (id)
+);
+
+CREATE TABLE invoices (
+  id INT GENERATED BY DEFAULT AS IDENTITY,
+  total_amount DECIMAL,
+  generated_at TIMESTAMP,
+  payed_at TIMESTAMP,
+  medical_histories_id INT REFERENCES medical_histories(id),
+  PRIMARY key (id)
+);
+
+CREATE TABLE treatements (
+  id INT GENERATED BY DEFAULT AS IDENTITY,
+  type VARCHAR,
+  name VARCHAR,
+  PRIMARY KEY (id)
+);
+
+CREATE TABLE invoice_items (
+  id INT GENERATED BY DEFAULT AS IDENTITY,
+  unit_price DECIMAL,
+  quantity INT,
+  invoice_id INT REFERENCES invoices (id),
+  treatement_id INT REFERENCES treatements (id),
```

Comment:
- [ ] [optional] @team it would be great if you considered correcting the typo error in treatments, this will ensure your association works.